### DISCUSSION TITLE
Converting HEIC Images in Ruby on Rails

### DISCUSSION THREAD

**Discussion by michalkorzawski** [therubyradar/submissions#105] [Submissions]
I wrote [Converting HEIC Images in Ruby on Rails](https://medium.com/expsoftwareengineering/converting-heic-images-in-ruby-on-rails-59242b16be7a) this week.

This is what it's about:
How to convert High Efficiency Image File Format (.HEIF / .HEIC) images in Ruby on Rails without a frontend library dependency.

This is why this submission will help/entertain our readers:
All others posts about this topic usually suggest using [Heic2any](https://github.com/alexcorvi/heic2any) library to preview heic images on the frontend and then adding a custom heic previewer on the backend. This, unfortunately, is very slow. I approach this issue from a different angle, converting heic images on the backend. This blog post shows how to do it.

**Comment by andrewmcodes** [comment]
There have been so many times in my life where I needed this exact post 👏
